handlers/test/unit/handlers/test_rpm.py

```diff
@@ -0,0 +1,352 @@
+from unittest import TestCase
+
+from mock import patch, Mock
+
+from pulp_rpm.handlers.rpm import (
+    PackageReport,
+    GroupReport,
+    PackageProgress,
+    PackageHandler,
+    GroupHandler)
+
+
+MODULE = 'pulp_rpm.handlers.rpm'
+
+
+class TestPackageReport(TestCase):
+
+    @patch(MODULE + '.ContentReport')
+    def test_succeeded(self, content_report):
+        details = {
+            'resolved': [1, 2, 3],
+            'deps': [4, 5]
+        }
+
+        # test
+        report = PackageReport()
+        report.set_succeeded(details)
+
+        # validation
+        content_report.set_succeeded(details, 5)
```

Comment:
I'll take your word for it that this will fail somehow if the right thing didn't happen. A comment to explain would probably be helpful. If this test failed, and I looked at it, I don't know if I'd have any idea what it's actually verifying.

Same below.